executors_test.go

```diff
@@ -532,6 +532,33 @@ func Test_Create_Non_PK_ID(t *testing.T) {
 	})
 }
 
+func Test_Embedded_Struct(t *testing.T) {
```

Comment:
I added this test to go through the whole CRUD flow, should therefore cover most parts.